changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor
```

As it doesn't introduce new functionality
```suggestion
- bump: patch
```